bin/lms-rtparser.php

```diff
@@ -670,7 +673,45 @@ public function doValidate(&$uri, $config, $context)
             }
         }
 
-        if (!$prev_tid) { // generate new ticket if previous not found
+        // generate new messege if previous ticket found or ticket was resolved
+        // less than rt.allow_modify_resolved_tickets_newer_than variable value seconds
+
+        if (($prev_tid && $prev_tid_contents['state'] != RT_RESOLVED) ||
+            ($prev_tid && $prev_tid_contents['state'] == RT_RESOLVED && ($prev_tid_contents['resolvetime'] + $modify_ticket_timeframe) > time())) {
+
+            // find userid
+            $requserid = $DB->GetOne(
+                "SELECT id FROM vusers WHERE email = ? AND email <> ''",
+                array($fromemail)
```

Comment:
A to $fromemail może być puste, że aż robisz warunek email <> ''?

Reply:
Ja tego elementu nie tykałem zamieniłem tylko kolejność warunków bo inaczej musiałbym brnąć w ich negacje co obniżyło by czytelność kodu ...